optimum/intel/pipelines/accelerator_utils.py

```diff
@@ -0,0 +1,215 @@
+        IPEXModelForTokenClassification,
+    )
+
+    IPEX_TASKS_MAPPING = {
+        "audio-classification": (IPEXModelForAudioClassification,),
+        "feature-extraction": (IPEXModel,),
+        "fill-mask": (IPEXModelForMaskedLM,),
+        "image-classification": (IPEXModelForImageClassification,),
+        "question-answering": (IPEXModelForQuestionAnswering,),
+        "summarization": (IPEXModelForSeq2SeqLM,),
+        "text2text-generation": (IPEXModelForSeq2SeqLM,),
+        "text-classification": (IPEXModelForSequenceClassification,),
+        "text-generation": (IPEXModelForCausalLM,),
+        "token-classification": (IPEXModelForTokenClassification,),
+        "translation": (IPEXModelForSeq2SeqLM,),
+        "zero-shot-classification": (IPEXModelForSequenceClassification,),
+    }
+
+else:
+    IPEX_TASKS_MAPPING = {}
+
+
+if is_openvino_available():
+    from ..openvino import (
+        OVModelForAudioClassification,
+        OVModelForAudioFrameClassification,
+        OVModelForAudioXVector,
+        OVModelForCausalLM,
+        OVModelForCTC,
+        OVModelForFeatureExtraction,
+        OVModelForImageClassification,
+        OVModelForMaskedLM,
+        OVModelForQuestionAnswering,
+        OVModelForSeq2SeqLM,
+        OVModelForSequenceClassification,
+        OVModelForSpeechSeq2Seq,
+        OVModelForTextToSpeechSeq2Seq,
+        OVModelForTokenClassification,
+        OVModelForVision2Seq,
+        OVModelForVisualCausalLM,
+        OVModelForZeroShotImageClassification,
+    )
+    from ..openvino.modeling_base import OVBaseModel
+
+    OV_TASKS_MAPPING = {
+        "audio-classification": (OVModelForAudioClassification,),
+        "audio-frame-classification": (OVModelForAudioFrameClassification,),
+        "audio-xvector": (OVModelForAudioXVector,),
+        "automatic-speech-recognition": (OVModelForCTC, OVModelForSpeechSeq2Seq),
+        "feature-extraction": (OVModelForFeatureExtraction,),
+        "fill-mask": (OVModelForMaskedLM,),
+        "image-classification": (OVModelForImageClassification,),
+        "image-text-to-text": (OVModelForVisualCausalLM,),
+        "image-to-text": (OVModelForVision2Seq,),
+        "question-answering": (OVModelForQuestionAnswering,),
+        "summarization": (OVModelForSeq2SeqLM,),
+        "text2text-generation": (OVModelForSeq2SeqLM,),
+        "text-classification": (OVModelForSequenceClassification,),
+        "text-generation": (OVModelForCausalLM,),
+        "text-to-audio": (OVModelForTextToSpeechSeq2Seq,),
+        "token-classification": (OVModelForTokenClassification,),
+        "translation": (OVModelForSeq2SeqLM,),
+        "zero-shot-image-classification": (OVModelForZeroShotImageClassification,),
+    }
+else:
+    OV_TASKS_MAPPING = {}
+
+
+def get_openvino_model_class(
+    task: str, config: Optional["PretrainedConfig"] = None, model_id: Optional[str] = None, **model_kwargs
+):
+    if task.startswith("translation_"):
```

Comment:
```suggestion
    if task.startswith("translation"):
```

Shouldn't it be this way?